_episodes/01-introduction.md

```diff
@@ -1,15 +1,116 @@
+> {: .solution}
+{: .challenge}
+
+To learn more about ESMValTool, you can look at the [documentation](https://docs.esmvaltool.org/en/latest/introduction.html), the [official website](https://www.esmvaltool.org/about.html), or the [overview paper](https://gmd.copernicus.org/articles/13/1179/2020/) in *Geoscientific Model Development*.
+
+## How does ESMValTool work
+
+The figure below shows the different components of ESMValTool. Most of the work is done in the 'core', which performs a number of preprocessor steps. Outside of the core we have some configuration files that specify things like *where to find the CMIP data*. The most important file however, is the *recipe* that specifies which preprocessor functions need to be applied to what data. The recipe also points to a diagnostic script that is executed after the preprocessor and performs a more specific analysis on the preprocessed data.
 
+![figure showing ESMValTool architecture]({{ page.root }}/fig/esmvaltool_architecture.png)
+
+> ## Discussion: (dis)advantages of this approach
+>
+> Discuss or think about the pros and cons of this architecture for a moment. Then unfold the box below to see our answers.
+>
+>
+> > ## See our thoughts
+> >
+> > - Streamlining common preprocessing steps ensures consistency in the algorithms being used and the way they are executed. This facilitates comparison and reproducibility.
+> > - Provenance and citation information can be tracked through the entire workflow.
+> > - The core builds upon the [iris](https://scitools.org.uk/iris/docs/latest/) package, which is quite strict in order to comply with CF conventions and minimize unexpected results.
+> > - Recipes are easy to read and share.
+> > - A collection of recipes and diagnostic scripts is shipped with ESMValTool, ready for re-use. Everyone can add to this collection.
+> > - The recipe format takes some getting used to and may be a bit less flexible then working on the datasets directly.
+> > - Features or dataset support that are missing from ESMValCore can be a limiting factor.
+> {: .solution}
+{: .discussion}
+
+## Community
+
+ESMValTool is built and maintained by an active community of scientists and engineers. Many of the interactions take place on GitHub. Here, we briefly introduce you to some of the most important pages.
+
+> ## Challenge: meet ESMValGroup
+>
+> Browse to [github.com/ESMValGroup](https://github.com/ESMValGroup). This is our 'organization' GitHub page. Have a look around. How many collaborators are there? Do you know any of them? You should see 2 main repositories: ESMValTool and ESMValCore. Clicking on them will take you to there. How many people have contributed to each of them? Can you also find out how many people have contributed to this tutorial?
+>
+> > ## Solution
+> >
+> > At the time of making this lesson, there were 138 members of ESMValGroup. 55 of them contributed to ESMValTool, 48 to ESMValCore. 52 (!) people contributed to this tutorial.
```

Comment:
I like that there are actual numbers given here, but I can also see that this might be a problem since it might not automatically update and therefore the numbers could be "old" pretty fast.

Reply:
Maybe we have to add a date here so that it is clear when we were actually checking the numbers.

Reply:
But maybe the statement "At the time of making this lesson..." might be enough.